src/components/BuilderItem/BuilderItem.tsx

```diff
@@ -0,0 +1,35 @@
+import { useTranslation } from 'react-i18next';
+
+import { Actions } from 'components';
+import {
+  StyledTitleBoldMedium,
+  StyledTitleMedium,
+  StyledTitleBoldSmall,
+} from 'styles/styledComponents';
+import theme from 'styles/theme';
+
+import { StyledBuilderItem, StyledImg, StyledCol, StyledActions } from './BuilderItem.styles';
+import { BuilderItemProps } from './BuilderItem.types';
+
+export const BuilderItem = ({ getActions, ...props }: BuilderItemProps) => {
+  const { t } = useTranslation('app');
+  const { name, description, img, count, withHover } = props;
+
+  return (
+    <StyledBuilderItem withHover={withHover}>
+      {img && <StyledImg src={img} alt={name} />}
+      <StyledCol>
+        <StyledTitleBoldMedium>{name}</StyledTitleBoldMedium>
+        <StyledTitleMedium>{description}</StyledTitleMedium>
+        {count && (
+          <StyledTitleBoldSmall sx={{ marginTop: theme.spacing(0.6) }}>
+            {count} {t('item', { count })}
```

Comment:
Why count variable uses here two times?
Do we have "item" element in translations?